packages/Uploader/stories/Testing.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import Uploader, { UploaderContext } from "../src/Uploader";
+
+const styles = {
+  table: css`
+    td {
+      max-width: 210px;
+      overflow: hidden;
+      padding: 8px;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+  `,
+  fileInput: css`
+    align-items: center;
+    border: 1px dashed blue;
+    border-radius: 6px;
+    box-shadow: ${tokens.popover.shadow};
+    display: flex;
+    height: 200px;
+    justify-content: center;
+    width: 300px;
+  `,
+};
+const propTypes = {
+  hasUploadButton: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasUploadButton: false,
+};
```

Comment:
No big deal, but I'd suggest _not_ adding `propTypes` to story components. It's just extra maintenance.